src/FastTags/LexerTypes.hs

```diff
@@ -77,8 +77,13 @@ mkAlexInput s trackPrefixes = AlexInput
     utf8BOM' = "\xFEFF"
 
 mkSrcPos :: FilePath -> AlexInput -> SrcPos
-mkSrcPos filename (AlexInput {aiLine, aiPrefix}) =
-    SrcPos { posFile = filename, posLine = aiLine, posPrefix = aiPrefix }
+mkSrcPos filename (AlexInput {aiInput, aiLine, aiPrefix, aiAbsPos}) =
+    SrcPos { posFile = filename
+           , posLine = aiLine
+           , posOffset = Offset aiAbsPos
+           , posPrefix = aiPrefix
+           , posText = aiPrefix <> (Text.takeWhile (/= '\n') aiInput)
```

Comment:
I think it would be nice to not populate this field if `aiTrackPrefixes` is `False`. I.e. when we're generating tags for vim - there's no reason to keep lots of lines around which will not be consumed in the end.